src/app/static/src/js/components/runWorkflow/workflowSummary/reportParameter.vue

```diff
@@ -0,0 +1,103 @@
+                            <h5>{{ report.name }}
+                                <span>
+                                    <info-icon size="1.2x"
+                                               stroke="grey"
+                                               v-tooltip="reportInfo(report.name)"
+                                               class="custom-class"/>
+                                    </span>
+                            </h5>
+                        </div>
+                        <div class="row">
+                            <div id="report-params" class="col-12 col-md-6 col-lg-4">
+                                <div class="single-workflow-summary-content parameters-bg-color d-flex wow fadeInLeft">
+                                    <div class="workflow-summary-text">
+                                        <span class="text-muted d-inline-block">Parameters</span>
+                                        <div v-if="paramSize(report)">
+                                            <p id="params" v-for="(value, key, index) in report.params" :key="index">{{ key }}: {{ value }}</p>
+                                            <div id="default-params" v-if="paramSize(report) > 0">
+                                                <a href="#collapseSummary"
+                                                   class="pt-2 d-inline-block small"
+                                                   data-toggle="collapse"
+                                                   aria-expanded="false"
+                                                   aria-controls="collapseSummary"
+                                                >Show default...</a>
+                                                <div id="collapseSummary" class="collapse">
+                                                    <p id="default-params-collapse" v-for="(value, key) in showDetails(report.params)">{{ key }}: {{ value }}</p>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div v-else><p>There are no parameters</p></div>
+                                    </div>
+                                </div>
+                            </div>
+                            <span class="d-inline-block"></span>
+                            <!--Dependencies boxes should go here, you might want to consider using slots or
+                             perhaps add html here directly instead of creating another component -->
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue"
+import {InfoIcon} from "vue-feather-icons";
+import {Dependency, WorkflowReportWithDependency} from "../../../utils/types";
+import {VTooltip} from "v-tooltip";
+
+interface Props {
+    dependencies: Dependency;
```

Comment:
It's confusing that this type is 'Dependency' (and prop name is 'dependencies') since these aren't dependencies but the actual reports in the summary (not their deps).  I don't think this component needs anything else in 'Dependency', so could this prop just be `reports: WorkflowReportWithDependencies[]`

Reply:
It will need dependencies when dependencies  component is added. We will not be having a different component for dependencies and even if we do we will have to add it to this component as a slot.